hack/docs/Adding_Detectors_external.md

```diff
@@ -116,8 +95,49 @@ For example, consider a hypothetical authentication endpoint that returns `200 O
    ```bash
    export TEST_SECRET_FILE=".env"
    ```
+The `.env` file should be in the new detector's directory like this:
+```
+├── tailscale
+│   ├── .env
+│   ├── tailscale.go
+│   └── tailscale_test.go
+```
+
+Now that a `.env` file is present, the test file can load secrets locally.
+
+Next, Update the tests with these test cases at minimum:
+1. Found and verified (using a credential loaded from the .env file)
+2. Found and unverified (determinately, i.e. the secret is invalid)
+3. Found and unverified (indeterminately due to timeout)
+4. Found and unverified (indeterminately due to an unexpected API response)
+5. Not found
```

Comment:
Do you think it's worth calling out that the generator will create these cases by default, and "adding" them really just means making any necessary changes, of which there might not be any?

Reply:
@rosecodym great suggestion! I think that's worth calling out for sure